daemon/daemon.cc

```diff
@@ -378,12 +378,13 @@ tr_rpc_callback_status on_rpc_callback(tr_session* /*session*/, tr_rpc_callback_
 
 tr_variant load_settings(char const* config_dir)
 {
-    auto app_defaults = tr_variant::make_map();
-    tr_variantDictAddStr(&app_defaults, TR_KEY_watch_dir, ""sv);
-    tr_variantDictAddBool(&app_defaults, TR_KEY_watch_dir_enabled, false);
-    tr_variantDictAddBool(&app_defaults, TR_KEY_watch_dir_force_generic, false);
-    tr_variantDictAddBool(&app_defaults, TR_KEY_rpc_enabled, true);
-    tr_variantDictAddBool(&app_defaults, TR_KEY_start_paused, false);
+    auto app_defaults_map = tr_variant::Map{ 5 };
```

Comment:
_nit_ for `size_t` literals, the codebase make it explicit that it's unsigned, e.g. `tr_variant::Map{ 5U }`

Reply:
7ac9c3d